tools/c7n_index/c7n_index/metrics.py

```diff
@@ -128,35 +145,47 @@ def get_indexer(config, **kwargs):
 @indexers.register('es')
 class ElasticSearchIndexer(Indexer):
 
-    def __init__(self, config, **kwargs):
+    def __init__(self, config):
         self.config = config
-        self.es_type = kwargs.get('type', 'policy-metric')
-
-        host = [config['indexer'].get('host', 'localhost')]
+        # self.es_type = kwargs.get('type', 'policy-metric')
+        host = [config['indexer'].get('host')]
         kwargs = {}
-        kwargs['connection_class'] = RequestsHttpConnection
+        kwargs['connection_class'] = elasticsearch.RequestsHttpConnection
 
         user = config['indexer'].get('user', False)
         password = config['indexer'].get('password', False)
         if user and password:
             kwargs['http_auth'] = (user, password)
 
-        kwargs['port'] = config['indexer'].get('port', 9200)
+        kwargs['port'] = config['indexer'].get('port')
 
-        self.client = Elasticsearch(
+        self.client = elasticsearch.Elasticsearch(
             host,
             **kwargs
         )
 
-    def index(self, points):
-        for p in points:
-            p['_index'] = self.config['indexer']['idx_name']
-            p['_type'] = self.es_type
-
-        results = helpers.streaming_bulk(self.client, points)
-        for status, r in results:
-            if not status:
-                log.debug("index err result %s", r)
+    def index(self, data):
+        # for backward compatibility with indexing docs from s3
+        # using an if statement to determine the caller
+        try:
+            if "sqs" in self.config:
+                res = self.client.index(
+                    index=data['policy']['resource'], doc_type=data['policy']['name'],
+                    body=data)
+                log.info("results: {}, index_created:{}, doc_type: {}".format(
+                    res, data['policy']['resource'], data['policy']['name']))
+                return res
+            else:
```

Comment:
No need for `else` and can outdent following lines.

Reply:
same as `localhost`